library/agent/InspectionStatistics.ts

```diff
@@ -176,6 +226,50 @@ export class InspectionStatistics {
     }
   }
 
+  onIPAddressMatches(matches: { key: IPListKey; monitor: boolean }[]) {
+    if (matches.length > 0) {
+      this.ipAddresses.total += 1;
+    }
+
+    const blockingMatches = matches.filter((match) => !match.monitor);
```

Comment:
Performance: Filter will loop over all elements, we could break on first match

Reply:
```suggestion
    const blockingMatch = matches.find(match) => !match.monitor);
```

Multiline suggestion does not work 😢

Reply:
See https://github.com/AikidoSec/firewall-node/pull/505/commits/2fa9007447fd85386d50db632c3677ce800c294e